fs/xfstests.py

```diff
@@ -196,10 +196,15 @@ def setUp(self):
             else:
                 packages.extend(['btrfs-progs-devel'])
 
+            packages_remove = ['indent', 'btrfs-progs-devel']
             if self.detected_distro.name == 'rhel' and\
                     self.detected_distro.version.startswith('8'):
-                packages.remove('indent')
-                packages.remove('btrfs-progs-devel')
+                packages = list(set(packages) - set(packages_remove))
+
+            packages_remove = ['dump', 'indent', 'btrfs-progs-devel']
```

Comment:
may be ideal way is just to update `packages_remove`  list variable adding dump package only inside a condition 

```

            if self.detected_distro.name == 'rhel' and\
                    self.detected_distro.version.startswith('9'):
                    packages_remove.append('dump')
                packages = list(set(packages) - set(packages_remove))

```

Reply:
@PraveenPenguin done changes, please review

[root@ltcden10-lp1 fs]# cat nohup.out 
JOB ID     : 24fcbbc25c7c7f231cff018da6ac9d593478e2a5
JOB LOG    : /home/disha/avocado-fvt-wrapper/results/job-2022-01-21T22.33-24fcbbc/job.log
 (1/1) xfstests.py:Xfstests.test:  FAIL: One or more tests failed. Please check the logs. (7319.25 s)
RESULTS    : PASS 0 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/disha/avocado-fvt-wrapper/results/job-2022-01-21T22.33-24fcbbc/results.html
JOB TIME   : 7329.50 s

[job.log](https://github.com/avocado-framework-tests/avocado-misc-tests/files/7923712/job.log)